### ISSUE TITLE
News cannot be deleted in Administration module

### ISSUE THREAD

**Issue by MediKathi** [georgringer/news#1019]
## Bug Report

**Current Behavior**
The click on the delete Record, while editing a newd record, button leads into
```
Not Found

The requested URL /typo3/undefined was not found on this server.
```

**Environment**
- TYPO3 version: 9.5.9
- news version: 7.2.3
- No Composer
- OS: Ubuntu 18.04.2 LTS

**Comment by georgringer** [comment]
You are talking about the admin module?

**Comment by MediKathi** [comment]
Yes, the Admin-Module in the Typo3 Backend.
Deleting a News-Record using the List module works, but not in the News-Admin module.

**Comment by angeloprevitali** [comment]
I can confirm the missing delete function.

TYPO3 version: 9.5.9
news version: 7.2.3
No Composer

**Comment by MediKathi** [comment]
It seams that this Bug was fixed with Typo3 9.5.14, can someone confirm this?

**Comment by georgringer** [comment]
confirmed, closing issue
